<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPlato::ConfigReportTemplatesPanel</class>
 <widget class="QWidget" name="KPlato::ConfigReportTemplatesPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QToolButton" name="insertModule">
     <property name="text">
      <string>Insert</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>libs/ui</normaloff>libs/ui</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>181</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="removeModule">
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff>libs/ui</normaloff>libs/ui</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QTreeView" name="reportTemplatesView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
